sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -196,6 +196,21 @@ The Azure Monitor Distro uses EventSource for its own internal logging. The logs
 OpenTelemetry also provides it's own [self-diagnostics feature](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/README.md#troubleshooting) to collect internal logs.
 An example of this is available in our demo project [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/OTEL_DIAGNOSTICS.json).
 
+**Missing Request Telemetry**
+
+If an app has a reference to the `OpenTelemetry.Instrumentation.AspNetCore` package, it could be missing request telemetry. To resolve this issue:
+
+* Either remove the reference to the `OpenTelemetry.Instrumentation.AspNetCore` package (or)
+* Add `AddAspNetCoreInstrumentation` to the OpenTelemetry TracerProvider configuration as per the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore).
+
+**Few or all Dependency Telemetries are missing**
```

Comment:
nit: I would change this to match the header above
```suggestion
**Missing Dependency Telemetry**
```

Reply:
Either partial or complete dependency telemetry may be missing. Having just one of the packages will result in partial telemetry, while referencing both without proper configuration can lead to no telemetry at all. I feel "Few or all Dependency Telemetries are missing" fits better here than "Missing Dependency Telemetry".